### ISSUE TITLE
Support nested yml format

### ISSUE THREAD

**Issue by khepin** [servergrove/TranslationEditorBundle#1]
If my files are formatted like this:

```
homepage:
    title: Welcome
    sidebar:
        title: Menu
        subtitle: Chose something to do

aboutpage:
    title: About us
    contact: Contact us
```

Then the bundle will import only 2 keys: `homepage` and `aboutpage` with values: `Array`. Either the import should flatten the file while importing them or it should recursively look for the values.

**Comment by pgodel** [comment]
correct. this format is not currently supported. Would you like to provide an implementation? :)

Ideally it should preserve the original format so it can be exported the same way.

**Comment by khepin** [comment]
I'm giving it a shot at least!

**Comment by khepin** [comment]
Quick question: are you guys opposed to including external js libraries for this?

**Comment by pgodel** [comment]
not at all.

**Comment by khepin** [comment]
Have a look at https://github.com/khepin/TranslationEditorBundle
on the "tests" branch (yeah, not my most clever moment, was in a hurry)!

I've added support for nested yml format as well as multiple domains. Added a command to delete the database as well if needed.

The nested format is fully supported in import and export, but only for edition right now, not possible yet to add new keys.
If different keys exist at any level of nesting, they're all merged in order to have one standard reference. This required changing the db organisation's quite a bit!

Added jquery and backbone to manage the view javascript part.

Let me know what you think. I can send a PR / wait for you to bombard me with bugs / wait til I add support for adding new keys.
